cypress/integration/features/editSnapshot.spec.js

```diff
@@ -0,0 +1,102 @@
+context('Edit snapshot', () => {
+  before(() => {
+    cy.task('createSnapshot', {
+      firstName: 'Phineas',
+      lastName: 'Flynn',
+      assets: [],
+      createdBy: 'Dat',
+      systemIds: [],
+      created: '2019-06-09T15:46:47.857Z',
+      dob: '2000-06-09',
+      vulnerabilities: [],
+      id: '1'
+    });
+
+    cy.setHackneyCookie(true);
+  });
+
+  after(() => {
+    cy.task('deleteSnapshot', '1');
+  });
+
+  describe('Edit snapshot', () => {
+    it('Displays editable snapshot if there are no assets, vulnerabilites and notes added', () => {
+      cy.visit(`http://localhost:3000/snapshots/1`);
+
+      cy.get('h1').should('contain', 'Phineas Flynn');
+
+      cy.get('h2').should('contain', 'Things to explore with the resident');
+
+      cy.get('[data-testid=accordion-item]')
+        .should('contain', 'Financial stability')
+        .and('contain', 'Physical health')
+        .and('contain', 'Mental health')
+        .and('contain', 'Behaviour and engagement')
+        .and('contain', 'Relationships and support network')
+        .and('contain', 'Life events and transitions');
+
+      cy.get('[data-testid=notes-input]').should(
+        'contain',
+        "Any other notes you'd like to add?"
+      );
```

Comment:
Interested to get other people's opinions, but personally I'm not a fan of testing that strings of text exist unless they're part of the functionality in some way (for example, makes sense to test the name is present). If the copy gets rewritten or reworded I wouldn't expect the tests to break.

In more complex UI projects this approach is made more difficult as you could have a localisation or content team writing all the strings and they are pulled into the project at build time etc.

Again - just my personal opinion though - so happy to hear others, or if there's a good reason to test the exact string vs just `should('exist')` :)

Reply:
Should exist sounds fair to me

Reply:
Nice 👌